tests/test_logout.py

```diff
@@ -85,3 +89,68 @@ def test_logout_with_multiple_validators(casper, funded_privkeys,
+                                                          logout_validator_via_unsigned_msg):
+    validator_index = induct_validator(funded_privkey, deposit_amount)
+    expected_end_dynasty = casper.dynasty() + casper.DYNASTY_LOGOUT_DELAY()
+
+    logout_validator_via_unsigned_msg(validator_index, funded_privkey)
+
+    assert casper.validators__end_dynasty(validator_index) == expected_end_dynasty
+
+
+def test_logout_from_withdrawal_address_with_signature(casper,
+                                                       funded_privkey,
+                                                       deposit_amount,
+                                                       induct_validator,
+                                                       logout_validator_via_signed_msg,
+                                                       assert_tx_failed):
+    validator_index = induct_validator(funded_privkey, deposit_amount)
+    expected_end_dynasty = casper.dynasty() + casper.DYNASTY_LOGOUT_DELAY()
+
+    logout_validator_via_signed_msg(validator_index,
+                                    funded_privkey,
+                                    funded_privkey)
+
+    assert casper.validators__end_dynasty(validator_index) == expected_end_dynasty
+
+
+def test_logout_from_non_withdrawal_address_without_signature(casper,
+                                                              funded_privkeys,
+                                                              deposit_amount,
+                                                              induct_validator,
+                                                              logout_validator_via_unsigned_msg,
+                                                              assert_tx_failed):
+    validator_key = funded_privkeys[0]
+    non_validator_key = funded_privkeys[1]
+    assert(validator_key != non_validator_key)
+
+    validator_index = induct_validator(validator_key, deposit_amount)
+
+    assert_tx_failed(lambda:
+                     logout_validator_via_unsigned_msg(validator_index,
+                                                       non_validator_key))
+
+
+def test_logout_from_non_withdrawal_address_with_signature(casper,
+                                                           funded_privkeys,
+                                                           deposit_amount,
+                                                           induct_validator,
+                                                           logout_validator_via_signed_msg,
+                                                           assert_tx_failed):
+    validator_key = funded_privkeys[0]
+    non_validator_key = funded_privkeys[1]
+    assert(validator_key != non_validator_key)
+
+    validator_index = induct_validator(validator_key, deposit_amount)
+    expected_end_dynasty = casper.dynasty() + casper.DYNASTY_LOGOUT_DELAY()
+
+    logout_validator_via_signed_msg(validator_index, validator_key,
```

Comment:
I prefer if we are going to multiline arguments in a function call to have each arg on a separate line.

My favorite syntax is:
```python
logout_validator_via_signed_msg(
    validator_index,
    validator_key,
    non_validator_key
)
```

But the following is cool too:
```python
logout_validator_via_signed_msg(validator_index,
                                validator_key,
                                non_validator_key)
```